processor/simple_supply_tp/payload.py

```diff
@@ -35,6 +35,9 @@ def data(self):
         if self._transaction.HasField('create_record'):
             return self._transaction.create_record
 
+        if self._transaction.HasField('transfer_record'):
```

Comment:
Should these also be based of the type? In other words, if it is `TRANSFER_RECORD`, but doesn't have a `'transfer_record'` field, it should be an invalid transaction.